src/Tizen.Applications.Common/Tizen.Applications/ApplicationInfo.cs

```diff
@@ -293,6 +315,36 @@ public bool IsPreload
             }
         }
 
+        /// <summary>
+        /// Gets the application's category.
```

Comment:
What value can I get?
There is not enough information about category

Reply:
I change the summry of Categories to
"Gets the application's category values specified in the tizen-manifest"